### ISSUE TITLE
Take exp for start a crate

### ISSUE THREAD

**Issue by Axilate18** [nulli0n/GoldenCrates#33]
When i try to put the condition of exp, i put it on 20 for exemple, and nothing happend like i don't have this condition.

**Comment by nulli0n** [comment]
It just takes specified amount of exp from a player.

**Comment by Axilate18** [comment]
Yes that the issue it's supposed to take exp from the player, but the crate open without take any exp

**Comment by nulli0n** [comment]
Works fine for me. More details, error logs?
